projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/RunJob.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+
+export interface IRunJobDialogProps {
+    /**
+     * Current Job path
+     */
+    jobPath: string;
+}
+
+
+export default class RunJobDialong extends Component<IRunJobDialogProps> {
+    /**
+   * Returns a React component for rendering a run menu.
+   *
+   * @param props - component properties
+   * @returns React component
+   */
+    constructor(props: IRunJobDialogProps) {
+        super(props);
+    }
+    /**
+    * Renders a dialog for running a data job.
+    *
+    * @returns React element
+    */
+    render(): React.ReactElement {
+        return (
+            <>
+                <div className='jp-vdk-input-wrapper'>
+                    <label className='jp-vdk-label' htmlFor="jobPath">Path to parent directory:</label>
+                    <input type="text" id="jobPath" className='jp-vdk-input' placeholder={this.props.jobPath} onChange={this._onPathChange} />
+                </div>
+                <div className='jp-vdk-input-wrapper'>
+                    <label className='jp-vdk-label' htmlFor="vdkArguments">Arguments:</label>
+                    <input type="text" className='jp-vdk-input' placeholder='{"key": "value"}' onChange={this._onArgsChange} />
+                </div>
+                <ul id="argumentsUl" className="hidden"></ul>
+            </>
+        );
+    }
+    /**
+   * Callback invoked upon changing the  job path input.
+   *
+   * @param event - event object
+   */
+    private _onPathChange = (event: any): void => {
+        const element = event.currentTarget as HTMLInputElement;
+        let value = element.value;
+        if(!value){
+            value = this.props.jobPath;
+        }
+        sessionStorage.setItem("job-path", value)
```

Comment:
I think using sessionStorage here is not a great idea

Reply:
I will change it to a variable

Reply:
You can also leave this if you like. 
I don't really see a better way of doing it.

Reply:
I tried to assign it to a variable and got errors, I will leave it like that for now